test/commands/index.test.js

```diff
@@ -25,5 +25,6 @@ describe("index command", function() {
         assert(stdout.output.startsWith("Develop and test Adobe Asset Compute workers"));
         assert(stdout.output.includes("asset-compute:run-worker   Run worker from local project using Docker"));
         assert(stdout.output.includes("asset-compute:test-worker  Run tests from local project"));
+        assert(stdout.output.includes("asset-compute:devtool      Runs the Asset Compute Developer Tool UI"));
```

Comment:
BTW, `Asset Compute Developer Tool UI` sounds like a mouthful...

Maybe drop the UI at the end: `Asset Compute Developer Tool`.

And it is interactive compared to the other commands, so maybe `Starts...` instead of `Runs...`